process-dir.js

```diff
@@ -2,51 +2,62 @@ let { dirname, join, sep } = require('path')
 let { promisify } = require('util')
 let lineColumn = require('line-column')
 let globby = require('globby')
-let rimraf = promisify(require('rimraf'))
 let fs = require('fs')
-
 let writeFile = promisify(fs.writeFile)
 let readFile = promisify(fs.readFile)
 let lstat = promisify(fs.lstat)
 let readdir = promisify(fs.readdir)
+let rimraf = promisify(require('rimraf'))
 
 const NAME = /(^|\n)(let\s+|const\s+|var\s+)(\S+|{[^}]+})\s*=/m
+const INDEX_JS = 'index.js'
+const INDEX_CJS = 'index.cjs'
+const BROWSER_JS = '.browser.js'
+const INDEX_BROWSER_JS = 'index.browser.js'
+const INDEX_NATIVE_JS = 'index.native.js'
+const PACKAGE_JSON = 'package.json'
+const OTHER_EXTENSIONS = ['types', 'style', 'styl', 'sass', 'less']
+const ENV_PROD = 'process.env.NODE_ENV === "production"'
+const ENV_NOT_PROD = 'process.env.NODE_ENV !== "production"'
 
-function error (msg) {
+const error = msg => {
```

Comment:
Code style changes are not cool

Reply:
I used arrow functions in the helpers, and the function declaration in the main __process()__ function. I thought it easier to read that way.